framework/src/org/checkerframework/framework/test/CheckerFrameworkPerDirectoryTest.java

```diff
@@ -71,7 +72,9 @@ public CheckerFrameworkPerDirectoryTest(
         this.testFiles = testFiles;
         this.checkerName = checker.getName();
         this.testDir = "tests" + File.separator + testDir;
-        this.checkerOptions = Arrays.asList(checkerOptions);
+        this.checkerOptions = new ArrayList<>();
+        this.checkerOptions.addAll(Arrays.asList(checkerOptions));
+        this.checkerOptions.add("-AprintErrorStack");
```

Comment:
On line 3 this file imports TestConfigurationBuilder.
There https://github.com/typetools/checker-framework/blob/master/framework/src/org/checkerframework/framework/test/TestConfigurationBuilder.java#L58
we already add this option.
Do we need to add it again here?